template/visual-diff/visual-diff.json

```diff
@@ -0,0 +1,5 @@
+{
+  "integrationFolder": "visual-diff/integration",
```

Comment:
This updates the paths in the Cypress config file to the correct location in both local and production builds